flytekit/configuration/internal.py

```diff
@@ -60,7 +60,7 @@ class GCP(object):
 
 class Credentials(object):
     SECTION = "credentials"
-    COMMAND = ConfigEntry(LegacyConfigEntry(SECTION, "command"), None, list)
+    COMMAND = ConfigEntry(LegacyConfigEntry(SECTION, "command", list), YamlConfigEntry("admin.command", list))
```

Comment:
Is this change correct, @wild-endeavor ?

Reply:
will take a look monday.  i remember specifically leaving it outside last time for some reason.